injectproxy/routes.go

```diff
@@ -205,11 +205,14 @@ func NewRoutes(upstream *url.URL, label string, opts ...Option) (*routes, error)
 
 func (r *routes) enforceLabel(h http.HandlerFunc) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		lvalue := req.FormValue(r.label)
-		if lvalue == "" {
+		req.ParseForm()
+		lvalues, present := req.Form[r.label]
+		if !present || len(lvalues) == 0 || lvalues[0] == "" {
```

Comment:
should we check if _every_ value is non-empty?

Reply:
Yes, that make sense. I'll update the code.

Reply:
Or maybe we just make sure, *not all* of the values are empty.

Reply:
I have edited the code. Now it checks *every* value, and reports error if one of the value is empty.